pkg/micropython/Makefile.include

```diff
@@ -2,6 +2,7 @@
 MP_RIOT_HEAPSIZE ?= 16384U
 
 CFLAGS += -DMP_RIOT_HEAPSIZE=$(MP_RIOT_HEAPSIZE)
+CFLAGS += -Wno-unused-parameter
```

Comment:
This globally disables this warning for the entire project when this package is included. One of the fixes suggested by @benpicco should be considered instead.